website/data/api-docs-nav-data-hidden.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
The `api-docs` and `docs` routes seemed appropriate to continue to keep hidden, to match previous behaviour. I've used the `_temp-enable-hidden-pages` fix to allow this.

Idea here is it'll be easy to keep track of "hidden" pages during this temporary fix stage, since they're in a separate `nav-data.json` file. And easy to move into the proper "visible" `nav-data.json` file if any of these pages should actually be visible in the sidebar.